### DISCUSSION TITLE
60GHz mmWave Fall Detection Pro | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#964] [Q&A]
# 60GHz mmWave Fall Detection Pro | Seeed Studio Wiki

60GHz mmWave Fall Detection Pro

https://wiki.seeedstudio.com/Radar_MR60FDA1/

**Comment by exanoelect** [comment]
Hi, 
Regarding to wiki above: universal-method---use-j-link-to-burn-firmware. I've purchase J-link segger V8 to update its firmware. But it cannot connect. How to solve this? Is it possible to?  Thank you.

Best regard.

**Reply by exanoelect** [reply]
Its succesfully downloaded, i was wrong in Jlink pin designator
